packages/@azure/servicebus/data-plane/lib/namespace.ts

```diff
@@ -125,11 +125,12 @@ export class Namespace {
   }
 
   /**
-   * Closes the AMQP connection created by this namespace along with AMQP links for sender/receivers
-   * created by the queue/topic/subscription clients created in this namespace.
+   * Closes the AMQP connection created by this ServiceBusClient along with AMQP links for
+   * sender/receivers created by the queue/topic/subscription clients created by this
+   * ServiceBusClient.
    * Once closed,
-   * - the namespace cannot be used to create anymore clients for queues/topics/subscriptions
-   * - the clients created in this namespace cannot be used to send/receive messages anymore
+   * - this ServiceBusClient cannot be used to create any new queues/topics/subscriptions clients.
+   * - the clients created by this ServiceBusClient cannot be used to send/receive messages anymore.
```

Comment:
Suggestion: Put the note about the clients created by this no longer being useful before the other note.

I think people expect that when they call close they won't be able to use this anymore, but the information about the clients no longer working is a bit more subtle so I think if we have it as the first bullet is might stand out a bit more.

Reply:
Done